WalletWasabi/Tor/Socks5/Pool/TorHttpPool.cs

```diff
@@ -345,6 +357,10 @@ private bool GetPoolConnectionNoLock(string host, ICircuit circuit, out TorTcpCo
 		{
 			Logger.LogTrace($"Beware! Too many active connections: '{string.Join(", ", hostConnections.Select(c => c.ToString()))}'.");
 		}
+		else
+		{
+			Logger.LogTrace($"There are active connections [{hostConnections.Count}]: '{string.Join(", ", hostConnections.Select(c => c.ToString()))}'.");
+		}
```

Comment:
This helps a lot with debugging. We can safely remove it though.

Reply:
I would say it's best to remove this log. It's helpful only for debugging of this particular bug I think. Also if we encounter a similar bug, we can add this line easily again.